src/applications/vaos/new-appointment/components/TypeOfCarePage/index.jsx

```diff
@@ -62,7 +62,10 @@ export default function TypeOfCarePage() {
       const covidLabel = (
         <>
           COVID-19 vaccine
-          <span className="usa-label vads-u-margin-left--1">New</span>
+          <span className="usa-label vads-u-margin-left--1" aria-hidden="true">
+            New
+          </span>
+          <span className="sr-only">New</span>
```

Comment:
I liked Peter's suggestion here: https://github.com/department-of-veterans-affairs/va.gov-team/issues/25383#issuecomment-866955852

If we put the sr-only text before `COVID-19 vaccine` and add some extra context, it could read out as "New type of care: COVID-19 vaccine," which I think is clearer than "COVID-19 vaccine new."

Reply:
Done.